dist/index.html

```diff
@@ -1 +1 @@
-<!doctype html><html lang="en"><head><meta charset="utf-8"><title>Skycoin Explorer</title><base href="/"><meta name="viewport" content="width=device-width,initial-scale=1"><link href="assets/styles.css" type="text/css" rel="stylesheet" media="screen,projection"/><link rel="icon" type="image/x-icon" href="favicon.ico"><link href="styles.69c3560ff48770e3059f.bundle.css" rel="stylesheet"/></head><body><app-root></app-root><script type="text/javascript" src="inline.1f30c6c19fd81468cb27.bundle.js"></script><script type="text/javascript" src="polyfills.b89f6da3f94d82c339a7.bundle.js"></script><script type="text/javascript" src="scripts.e0f4a942c9ea3a773769.bundle.js"></script><script type="text/javascript" src="main.599200bfe7034cd8195e.bundle.js"></script></body></html>
\ No newline at end of file
+<!doctype html><html lang="en"><head><meta charset="utf-8"><title>Skycoin Explorer</title><base href="/"><meta name="viewport" content="width=device-width,initial-scale=1"><link href="assets/styles.css" type="text/css" rel="stylesheet" media="screen,projection"/><!--TODO: Fix icon--><link rel="icon" type="image/x-icon" href="skycoin.ico"><link href="styles.23e97c0364f2d7f9207c.bundle.css" rel="stylesheet"/></head><body><app-root></app-root><script type="text/javascript" src="inline.21b387dd6c507e5d738f.bundle.js"></script><script type="text/javascript" src="polyfills.b89f6da3f94d82c339a7.bundle.js"></script><script type="text/javascript" src="scripts.e0f4a942c9ea3a773769.bundle.js"></script><script type="text/javascript" src="main.06b2da069750fd0665ad.bundle.js"></script></body></html>
```

Comment:
Please dont create a new build in your pull request

Reply:
Sorry for the late response. The change seems to have been for an update of the branch that I did not do correctly, the Index file is now unmodified.